AppAuth.podspec

```diff
@@ -44,8 +47,9 @@ It follows the OAuth 2.0 for Native Apps best current practice
 
   # Subspec for the full AppAuth library, including platform-dependant external user agents.
   s.subspec 'ExternalUserAgent' do |externalUserAgent|
-
-    externalUserAgent.source_files = "Source/*.{h,m}", "Source/AppAuthCore/*.{h,m}", "Source/AppAuth/*.{h,m}"
+    externalUserAgent.dependency 'AppAuth/Core'
+    
+    externalUserAgent.source_files = "Source/AppAuth.h", "Source/AppAuth/*.{h,m}"
```

Comment:
Not for this PR, but I feel like this directory should probably be renamed `ExternalUserAgent`

Reply:
Should this go in a future PR of mine?

Reply:
Yes, have a separate PR for the directory name change.